test/api/search_controller_test.rb

```diff
@@ -80,4 +80,45 @@ class SearchControllerTest < ActionDispatch::IntegrationTest
 
     puts "#{File.basename(__FILE__)}: #{self.method_name} successful!"
   end
+
```

Comment:
sweet, move to new file bulk_download_service_test.rb

Reply:
f5a85e5